libtiledbsoma/src/soma/soma_transformers.h

```diff
@@ -8,7 +8,9 @@ namespace tiledbsoma {
 
 class OutlineTransformer : public Transformer {
    public:
-    OutlineTransformer(SOMACoordinateSpace coordinate_space);
+    OutlineTransformer(
```

Comment:
Could you add some docs either here or above the `columns` private member on what `columns` is and how it is used?